eng/pipelines/docindex.yml

```diff
@@ -5,7 +5,8 @@ jobs:
 
   - job: UpdateDocsMsBuildConfig
     pool:
-      vmImage: ubuntu-20.04
+      name: azsdk-pool-mms-ubuntu-2004-general
+      vmImage: MMSUbuntu20.04Compliant
```

Comment:
Do we need to add the word Compliant at the end here?

Reply:
That's what we have in the pool.
![image](https://user-images.githubusercontent.com/48036328/198690176-75aec9fd-25dd-4100-b037-e8aecceca201.png)

Reply:
@mikeharder do you think we should stick with the image name we have configured in the pool? Or since we are only allowed one image per pool should we use a general name like "windows" for the vmImage name in our yml files?

Reply:
@sima-zhu in the meantime yes keep the name matching the image.

Reply:
I have this `vmImage` as convention. Based on my testing, it is ok to use pool name only without specifying `vmImage` name.